types/node-forge/index.d.ts

```diff
@@ -107,11 +107,66 @@ declare module "node-forge" {
             extensions: any[];
             publicKey: any;
             md: any;
+            setSubject:
+            /**
+              * Sets the subject of this certificate.
+              *
+              * @param attrs the array of subject attributes to use.
+              * @param uniqueId an optional a unique ID to use.
+              */
+            (attrs: any, uniqueId?: any) => {};
```

Comment:
Define functions like the rest of the file.  Put a return type if possible, and stricter parameter types as well.

Reply:
Add some "tests" for this new functionality.

Reply:
I am a typecript novice, can you give an example of these two requirements?

Reply:
Line 103 above is a good example.
```ts
addField(attr: CertificateField): void;
```
Something similar for this method, although untested and using an educated guess, may be:
```ts
setSubject(attrs: string[], uniqueId?: number): void;
```

As for "testing," just provide some example usage in the `node-forge-tests.ts` in this directory and make sure it compiles.

Reply:
ok, thank you!

Reply:
done, pls check.